### ISSUE TITLE
Severe redraw on 1.71.1

### ISSUE THREAD

**Issue by sicwan** [goatfungus/NMSSaveEditor#1022]
![image](https://github.com/user-attachments/assets/9f109d8e-3cfd-46a5-be3b-8380e9ae551b)

I'm getting a lot of redraw with the latest version of the save editor.  I've tried the follow but it doesn't help at all.
-Dsun.java2d.opengl=true -Dsun.java2d.d3d=false -Dsun.java2d.noddraw=true

**Comment by Sup1mSam** [comment]
Experiencing the same issue, which makes doing any editing practically impossible.

**Comment by jdm12983** [comment]
I don't know if it's a style/"theme" issue or what - I am not having that issue.

I tried with both light and dark themes.

**Comment by Barerwolf** [comment]
yeah its doing it to me too, using light theme so its def not a theme issue

**Comment by jdm12983** [comment]
Is your java updated to the latest version? Maybe that causes it? I doubt it, but something to look at.

Also, what are you doing to cause the re-draw like that? I will try more to see if I get it to happen eventually.

**Comment by sicwan** [comment]
yes my java is up to date, I even use another java app that used to have this issue, but its been working fine for about a year now

**Comment by jdm12983** [comment]
Yeah, I'll keep using the program -- see if I get the issue too. So far; it's working fine for me.  Hopefully it will be something that can be sorted out quickly for others here though.

Oh, one other thing: what version of Windows are you people using? I am on Windows 10.

**Comment by Barerwolf** [comment]
I doubt the exact version of Windows makes a difference but I'm on win11 pro

**Comment by Barerwolf** [comment]
The version before the world's update for editor didn't do this to me either btw so I'm not sure what the problem is

**Comment by Quaggmar** [comment]
I just tried both Light and Dark mode and am not getting any redraw issues. 

I do recall upon initially launching any of my game saves on the main screen that a Java update happened, in fact a rather significant several Java updates which kicked off as soon as i selected ALL of my saves. That is to say each individual save launched the same series of Java updates before i could access the saved data for editing.

Not sure what this means for you but perhaps running a manual Java Update may help you there.

**Comment by SingularityAnon** [comment]
Im on Win11 pro and getting it severely too. i know people have already chimed in just wanted to add mine

**Comment by jdm12983** [comment]
I see several people saying they are having redraw issues and on Windows 11; I haven't had this issue and on Windows 10.

Maybe coincidentally; but something for the creator of the save editor to look into.

**Comment by statopre** [comment]
Posted the same months ago. Conclusion was indeed, that it was a Windows 11 issue

**Comment by hehe-yay** [comment]
Windows 11 Pro here. Lastest Java updates. Crazy redraw issues.

**Comment by Barerwolf** [comment]
yeah i dont think theme has anything to do with it... just tried dracula didn't change its def a issue between win 11 os and java

**Comment by Barerwolf** [comment]
yo i figured it out on my end, for me it was the recording software "medal" that caused it over lap on nmssaveeditor

**Comment by sicwan** [comment]
I dont have any recording software enabled on my PC (or running I should say) I dont think its a windows 11 issue w/ java, the other java program I use had the issue, but they fixed it last year, and it no longer has the issue

**Comment by Barerwolf** [comment]
maybe click on show hidden icons on the bottom right of taskbar and shut down program one after another till it stops?, restart nmssaveeditor everytime a program it shut down

**Comment by SingularityAnon** [comment]
> Has the same problem on win10 and 11 . Issue fixed on this , and an other java app i had the same issue , by closing riva tuner server and msi afterburner.

This Worked for me!

**Comment by sicwan** [comment]
I did some more testing, and I got it to work w/o issues using this, 
"C:\Program Files\Common Files\Oracle\Java\javapath\javaw.exe" -Xmx4096M -Dsun.java2d.d3d=false -jar NMSSaveEditor.jar
I removed the -autoupdate from it as it errored out as not being a valid command, works even w/ MSI afterburner and Riva TNT Tuner running

**Comment by epiloso** [comment]
> I did some more testing, and I got it to work w/o issues using this, "C:\Program Files\Common Files\Oracle\Java\javapath\javaw.exe" -Xmx4096M -Dsun.java2d.d3d=false -jar NMSSaveEditor.jar I removed the -autoupdate from it as it errored out as not being a valid command, works even w/ MSI afterburner and Riva TNT Tuner running

Holy cow, this worked for me- Win11 Pro, updated Java. Thanks!

To clarify, I edited the _NMSSaveEditor.Bat_ file with this line (kept autoaupdate, seems to still work with that)
`java -Xmx4096M -Dsun.java2d.d3d=false -jar NMSSaveEditor.jar -autoupdate`

**Comment by curlymoe** [comment]
"To clarify, I edited the NMSSaveEditor.Bat file with this line (kept autoaupdate, seems to still work with that)
java -Xmx4096M -Dsun.java2d.d3d=false -jar NMSSaveEditor.jar -autoupdate"
Brilliant, this worked. I've been struggling with this for a long time, I dual boot Win 10 and 11, using Win 10 it has always been Ok, but as soon as I installed Win 11 this problem started. Many thanks to you and "sicwan" for finding the answer.
